tutorials/xr/deploying_to_android.rst

```diff
@@ -0,0 +1,68 @@
+.. _doc_deploying_to_android:
+
+Deploying to Android
+====================
+
+Setup
+------------
+Most mobile headsets run on Android and OpenXR support is making its way to these platforms. There are specific requirements for deploying XR projects to Android.
+For general requirements around exporting to Android, please first read :ref:`doc_exporting_for_android`.
+
+.. note::
+	Official support for the Android platform wasn't added to the OpenXR specification initially resulting in various vendors creating custom loaders to make OpenXR available on their headsets.
+	While the long term expectation is that all vendors will adopt the official OpenXR loader, for now these loaders need to be added to your project.
+
+Custom Android build
+--------------------
+In order to deploy the loaders into your project you will need to setup a custom Android build.
+
+Select **Install Android Build Template...** from the **Project** menu:
+
+.. image:: img/android_custom_build.png
+
+This will create a folder called **android** inside of your project that contains all the runtime files needed on Android. You can now customise this installation. Godot won't show this in the editor but you can find it with a file browser.
```

Comment:
```suggestion
This will create a folder called **android** inside of your project that contains all the runtime files needed on Android. You can now customize this installation. Godot won't show this in the editor but you can find it with a file browser.
```